ui/app/i18n/registration/locale_en.json

```diff
@@ -113,5 +113,10 @@
   "DELETE_LABEL": "Delete",
   "EDIT_LABEL": "Edit",
   "REGISTRATION_LABEL_RELATIONSHIP_EXISTS": "Given relationship already exists",
-  "PERSON_ATTRIBUTE_TYPE_PHONE_NUMBER": "Phone Number"
+  "PERSON_ATTRIBUTE_TYPE_PHONE_NUMBER": "Phone Number",
+  "REGISTRATION_LABEL_SIMILAR_PATIENT_FOUND":"Similar patient found",
```

Comment:
@IipperMangesh I would suggest using `similar patient(s) found` here (to cater for both plural and singular). Also please note the first 's' is in lower caps (to avoid the upper caps in the middle of the sentence, like in `2 Similar ..`)

Reply:
Yes, changed its starting letter 'S' to 's'.